### PR TITLE
Create ValidParentheses.java

### PR DESCRIPTION
**Pull request by adjamalbek**
- [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Java/blob/master/CONTRIBUTING.md).
- [x] This pull request is all my own work -- I have not plagiarized it.
- [x] All filenames are in PascalCase.
- [x] All functions and variable names follow Java naming conventions.
- [x] All new algorithms have a URL in their comments that points to Wikipedia or other similar explanations.

### CODE REVIEW

**Comment by skinox** [comment]
Can you reply to this, since I don't really understand how it works, there are a lot of pull requests, however only random one's are chosen and I see there are some pull requests that are 20+ days old/

**Review by adjamalbek — COMMENTED** [review]
updated the class

**Review by adjamalbek — COMMENTED** [review]
updated class

**Review by debasishbsws — COMMENTED** [review]
Add some JUnit tests

**Comment by adjamalbek** [comment]
> Add some JUnit tests

I did, https://github.com/TheAlgorithms/Java/pull/3709 2 days ago, for all the commits

File: src/main/java/com/thealgorithms/strings/ValidParentheses.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
+//        Given a string s containing just the characters '(', ')', '{', '}', '[' and ']', determine if the input string is valid.
```
Comment (debasishbsws) [inline src/main/java/com/thealgorithms/strings/ValidParentheses.java:1]:
```suggestion
package com.thealgorithms.strings;
//        Given a string s containing just the characters '(', ')', '{', '}', '[' and ']', determine if the input string is valid.
```

**Review by debasishbsws — CHANGES_REQUESTED** [review]
Build failing, need PAckage declaration

**Comment by adjamalbek** [comment]
All the errors are fixed

**Comment by debasishbsws** [comment]
unable to build.

**Comment by adjamalbek** [comment]
Sorry, I didn't put last commit

**Comment by adjamalbek** [comment]
Requesting review from @siriak and @yanglbme and @debasishbsws for approval and merging.

**Review by adjamalbek — COMMENTED** [review]
updated

**Review by siriak — APPROVED** [review]
Looks good, thanks!

**Comment by siriak** [comment]
@debasishbsws please review again

**Review by debasishbsws — DISMISSED** [review]
Nice, thank you for contributing

**Review by siriak — APPROVED** [review]

**Pull request merged**